lib/OpenQA/Schema/Result/Comments.pm

```diff
@@ -103,7 +103,7 @@ sub bugrefs ($self) { find_bugrefs($self->text) }
 Returns label value if C<$self> is label, e.g. 'label:my_label' returns 'my_label'
 =cut
 sub label ($self) {
-    return first { $_ !~ /^force_result/ } @{find_labels($self->text)};
+    return first { $_ } @{find_labels($self->text)};
```

Comment:
This could then simply return the first element of the array, there's no need for the `{ $_ }`-check.

Reply:
Oh yeah true